gulpfile.ts

```diff
@@ -22,14 +24,33 @@ interface APIModel extends GithubObject {
   outputPath: string
 }
 
+// configure dotenv and get github token
+dotenv.config()
```

Comment:
Better to put this in the npm script:

```
ts-node -r dotenv/config ...
```

That way we can optionally skip `.env` file reading, if necessary.